packages/forms/src/UIForm/UIForm.container.js

```diff
@@ -5,28 +5,87 @@ import React from 'react';
 import UIFormTranslatedComponent from './UIForm.component';
 import { formPropTypes } from './utils/propTypes';
 
+/**
+ * add error object on a formSchema if it doesn't exist
+ * @param {FormSchema} formSchema
+ * @return {FormSchema}
+ */
+function addErrorObject(formSchema) {
+	if (!formSchema.errors) {
+		return { errors: {}, ...formSchema };
+	}
+	return formSchema;
+}
+
+/**
+ * reinit liveState with empty errors if it doesn't exist
+ * @param {FormSchema} newFormSchema
+ * @param {State} prevState
+ * @return {State}
+ */
+const reinitLiveState = newFormSchema => () => ({
+	liveState: addErrorObject(newFormSchema),
+});
+
+/**
+ * update live state with new properties derived from user interacting
+ * with the form
+ * @param {Object} properties
+ * @param {State} prevState
+ * @return {State}
+ */
+const change = properties => prevState => ({
```

Comment:
As you externalized those pieces of code, I think you should give them more explicit names, explaining what they do instead of when they are called.

For example:
reinitLiveState 👍 
change --> setLiveStateProperties
setErrors --> setLiveStateErrors
submit --> setLiveAsInitialState